src/clientdlg.cpp

```diff
@@ -858,6 +858,16 @@ void CClientDlg::OnNumClientsChanged ( int iNewNumClients )
     SetMyWindowTitle ( iNewNumClients );
 }
 
+void CClientDlg::OnOpenGeneralSettings()
+{
+    ShowGeneralSettings ( SETTING_TAB_AUDIONET );
+}
+
+void CClientDlg::OnOpenMusicianProfileDialog()
```

Comment:
Similarly, now there's no dedicated profile dialog, this should change, at least to end `...Settings`.